### PR TITLE
Tweak README.md

### PR DESCRIPTION
**Pull request by XhmikosR**
* switch to shields.io for all badges
* use `const` in the JS snippets
* fix a http link and a redirect
* remove moot spaces around brackets
* fix syntax highlighting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,15 +149,15 @@ exports.buildStyles = buildStyles;
...
 
 Then, you must explicitly set that compiler in your gulpfille. Instead of setting a `compiler` prop on the `gulp-sass` instance, you pass the compiler into a function call when instantiating `gulp-sass`.
 
 These changes look something like this:
 
-``` diff
+```diff
 - var sass = require('gulp-sass'));
 - var compiler = require('sass');
 - sass.compiler = compiler;
```
Comment (xzyfer) [inline README.md:163]: const here too?

Comment (XhmikosR) [reply README.md:163]: this is a diff so I wasn't sure if I should change it...

Comment (xzyfer) [reply README.md:163]: Keeping consistent is good. There'll be lots of JS folks who have never been exposed to `var` before.

Comment (XhmikosR) [reply README.md]: Not sure if it's valid, but I changed these `var`s to `const`.

**Pull request merged**